src/server/typesMap.json

```diff
@@ -22,6 +22,15 @@
         "References": {
             "match": "^(.*\\/_references\\.js)$",
             "exclude": [["^", 1, "$"]]
+        },
+        "Datatables.net": {
+            "match": "^.*\\/(jquery\\.)?dataTables(\\.all)?(\\.min)?\\.js$",
+            "types": ["datatables.net"]
+        },
+        "Ace": {
+            "match": "^(.*)\\/ace.js",

```

Comment:
Please note that ace.js has a very generic folder name, ranging between "src", "src-min", "src-noconflict", etc. thus the need to match any folder name as a user is very likely to append the name.

In the data analyzed a user use the name "ace-src-min".

Reply:
It's interesting looking at the repo at https://github.com/ajaxorg/ace-builds . I see multiple folders with different builds all including ace.js, so I get why this is needed. Does the safelist mechanism detect and exclude multiple folders for the same filename? It would be good to (have a) test to make sure. cc @RyanCavanaugh .

Reply:
It does, I tested manually  having an src-min folder and on another path src-noconflict, and it excluded both of the folders. I will work on an automated test on another PR.